src/commands/cmd_help.nim

```diff
@@ -520,8 +526,10 @@ proc runChalkHelp*(cmdName = "help") {.noreturn.} =
         toOut += con4mRuntime.getHelpTopics()
       of "builtins":
         toOut = con4mRuntime.getBuiltinsTableDoc()
+      of "changelog", "release", "releases":
+        toOut += getEmbeddedDoc("changelog")
       else:
-        let toCheck = [arg, "core-" & arg, "howto-" & arg, "guide" & arg]
+        let toCheck = [arg, arg & ".md" & "core-" & arg, "howto-" & arg, "guide" & arg]
```

Comment:
are you missing a comma `,` here? i think this line should be as follows correct? With a `,` after the new item of `arg & ".md"` you are looking to match on ?

```
let toCheck = [arg, arg & ".md", & "core-" & arg, "howto-" & arg, "guide" & arg]
```

Reply:
good spot. thanks. updated